src/test/no_name_output/1/model.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?>
+<net name="no_name_output" version="11">
+	<layers>
+		<layer id="1" name="input1" type="Parameter" version="opset1">
+			<data shape="1,10" element_type="i8" />
+			<output>
+				<port id="0" precision="I8" names="input1">
+					<dim>1</dim>
+					<dim>10</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="0" name="input2" type="Parameter" version="opset1">
```

Comment:
Can we use name "input_1" and "input_2" to follow the convention?

Reply:
yes, but this is barely a test model ;-)